### PR TITLE
WinMisc: Update detection to flag XP as an unsupported OS.

### PR DESCRIPTION
**Pull request by MrCK1**
This is just a small update to WinMisc to bring it in line with previous changes relating to XP being dropped since 1.4.0. 

If anybody is still on XP for whatever reason, the console will graciously let them know that their OS is officially unsupported by us.

### CODE REVIEW

**Comment by atomic83GitHub** [comment]
Windows server 2003 , XP pro 64bit and ReactOS are also concerned by this PR?

**Comment by MrCK1** [comment]
ReactOS would appear as an unsupported OS anyways, because it's not a Microsoft Product. Windows Server 2003 and Win2K are the same "Major Version" category as XP.

Just so you know, this _doesn't actually stop those OSes from running PCSX2,_ it just flags them as `Unsupported Operating System` in the console. WxWidgets requires Windows NT as a minimum anyways, so that's the real limiting factor for us supporting older OSes.

File: common/src/Utilities/Windows/WinMisc.cpp [hunk 1]
```diff
@@ -239,32 +227,7 @@ wxString GetOSVersionString()
 					retval += L"Standard Edition";
 			}
 		}
```
Comment (turtleli) [inline common/src/Utilities/Windows/WinMisc.cpp]: So... why did you decide to keep this code?

**Comment by MrCK1** [comment]
I tried to fix it the easy way by running clang-format on the file, but I guess I can't do that...

I'll try to do a proper rebase, but this might take a while until I remember the steps...

**Pull request closed without merging**